diff --git a/oauthenticator/azuread.py b/oauthenticator/azuread.py
index be144c5..46b581f 100644
--- a/oauthenticator/azuread.py
+++ b/oauthenticator/azuread.py
@@ -81,7 +81,7 @@ class AzureAdOAuthenticator(OAuthenticator):
         id_token = resp_json['id_token']
         decoded = jwt.decode(id_token, verify=False)
 
-        userdict = {"name": decoded[self.username_claim]}
+        userdict = {"name": 'jupyter-'+decoded[self.username_claim].split('@')[0]}
         userdict["auth_state"] = auth_state = {}
         auth_state['access_token'] = access_token
         # results in a decoded JWT for the user data
